script.js

```diff
@@ -18,3 +18,63 @@ toggleButtonOff.addEventListener('click', () => {
 function closeMenu(toggleButton) {
   navbarLinks.classList.toggle('active')(bodyColor.classList.toggle('change')(toggleScroll.classList.toggle('fixed')));
 }
+
+const toggleTonicPop = document.getElementsByClassName('project-btn')[0];
+const tonicPopup = document.getElementsByClassName('tonic-popup')[0];
+const tonicPopClose = document.getElementsByClassName('toggle-popup')[0];
+const popupBlur = document.getElementsByClassName('header-section')[0];
+const blurNav = document.getElementsByClassName('toolbar')[0];
+
+toggleTonicPop.addEventListener('click', () => {
+  tonicPopup.classList.toggle('show')(popupBlur.classList.toggle('blur')(blurNav.classList.toggle('dim')));
+});
+
+tonicPopClose.addEventListener('click', () => {
+  if (tonicPopup.classList.toggle('show')(popupBlur.classList.toggle('blur')(blurNav.classList.toggle('dim')))) {
+    tonicPopClose();
+  }
+});
+
+const toggleMultiPop = document.getElementsByClassName('project-btn2')[0];
+const multiPopup = document.getElementsByClassName('multi-popup')[0];
```

Comment:
- [ ] Kindly use an array of objects to store Javascript objects containing information for each project.

<img width="1281" alt="Screenshot 2022-12-22 at 06 26 27" src="https://user-images.githubusercontent.com/81444338/209156799-3fa4db37-1328-4916-a017-7cc5e036c00b.png">